src/authentication/AuthenticationSection.tsx

```diff
@@ -198,6 +213,17 @@ export const AuthenticationSection = () => {
               loader={loader}
               ariaLabelKey="authentication:title"
               searchPlaceholderKey="authentication:searchForEvent"
+              toolbarItem={
+                <ToolbarItem>
+                  <Button
+                    onClick={() =>
+                      history.push(`/${realm}/authentication/create`)
+                    }
+                  >
```

Comment:
This `Button` can be a React Router `Link` by providing the `component` prop.
```suggestion
                  <Button component={Link} to={`/${realm}/authentication/create`}>
```